src/Widgets/GoogleMapsWidget/GoogleMapsWidgetComponent.js

```diff
@@ -109,6 +63,17 @@ function StaticGoogleMap({ address, apiKey, zoom, children }) {
   );
 }
 
+function scaleMapSize(width, height) {
+  if (width > maxWidth) {
```

Comment:
* [ ] FYI the input can be `null` according to line 43. This function works by change with `null` input, but we may want to make it more explicit. Also, with `0` input the function may output `NaN` values. I think with a guard like `!height || !width` we would avoid any potential headache.